### PR TITLE
New labor

### PR DESCRIPTION
**Pull request by knoiva-indecon**
Updated package tar.gz file since previous build had a small issue.

### CODE REVIEW

File: FrEDI/DESCRIPTION [hunk 1]
```diff
@@ -1,6 +1,6 @@
 Package: FrEDI
 Title: The Framework for Evaluating Damages and Impacts (FrEDI)
-Version: 2.0
+Version: 2.1.1
```
Comment (cahartin) [inline FrEDI/DESCRIPTION:3]: @knoiva-indecon let's make this 2.2.0

File: FrEDI/R/aggregate_impacts.R [hunk 2]
```diff
@@ -780,8 +781,8 @@ aggregate_impacts <- function(
   ###### Order Columns ######
   ### Order the data frame and ungroup
   ### Column indices of columns used in ordering
-  return_names  <- df_return %>% names
-  orderColNames <- c(groupByCols, "year") %>% (function(y){y[which(y %in% return_names)]})
+  return_names  <- df_return %>% names #; "got here9" %>% print
```
Comment (cahartin) [inline FrEDI/R/aggregate_impacts.R:784]: @knoiva-indecon  is this a comment?

File: FrEDI/install/00_install.R [hunk 3]
```diff
@@ -38,9 +38,51 @@ devtools::build_vignettes(pkg = package_location)
 ###### - Build Package but don't include vignettes
 devtools::build(pkg=package_location)
 
+###### Copy New Package ######
+### Copy package
+packageFile_dir    <- package_location %>% file.path( "..")
+packageFile_name   <- "FrEDI_2.1.1.tar.gz"
```
Comment (cahartin) [inline FrEDI/install/00_install.R:44]: update to 2.2.0

**Review by cahartin — CHANGES_REQUESTED** [review]
@knoiva-indecon a few minor comments.
Also can we update this 2.2.0?

File: FrEDI/install/00_install.R (hunk 3 above)
Comment (knoiva-indecon) [reply FrEDI/install/00_install.R]: Resolved in new push

File: FrEDI/R/aggregate_impacts.R (hunk 2 above)
Comment (knoiva-indecon) [reply FrEDI/R/aggregate_impacts.R]: Resolved in new push

File: FrEDI/DESCRIPTION (hunk 1 above)
Comment (knoiva-indecon) [reply FrEDI/DESCRIPTION]: Updated in new push

**Pull request merged**